proto/regen/ecocredit/v1/query.proto

```diff
@@ -644,3 +696,61 @@ message BatchBalanceInfo {
   // cancelled, updated with a reduced quantity, or processed.
   string escrowed_amount = 5;
 }
+
+// ClassCreatorInfo is the human-readable class creator information.
+message ClassCreatorInfo {
+  // address is the address that is allowed to create credit classes
+  string address = 1;
+}
```

Comment:
I don't think we need a separate proto message for this. We can just include `string address` in the response.

Reply:
Or in this case `repeated string address`.